Resources/Prototypes/Catalog/Cargo/cargo_materials.yml

```diff
@@ -45,3 +45,15 @@
   cost: 6000
   category: Materials
   group: market
+
+- type: cargoProduct
+  name: "crate of plasma glass sheets"
+  id: Materialplasmaglass
```

Comment:
This ID doesn't follow our [naming guidelines](https://hackmd.io/Awj5TvdXTaOmhU2XZOsdQg#Naming).